README.md

```diff
@@ -5,3 +5,7 @@
 
 ## vector-embedding.py
     demonstrates Vector embedding phase of LLM pipeline ie phaseII
+
```

Comment:
Sections should follow the same format (## Section Name), and empty lines between sections make the document easier to read.